packages/cli/internal/signatures.js

```diff
@@ -1,7 +1,4 @@
-function getFullFunctionSignature(contractName, functionName, functionParameters) {
-  const abi = hre.deployer.deployment.abis[contractName];
-  const functionAbi = abi.find((abiItem) => abiItem.name === functionName);
-
+function getFullFunctionSignature(functionAbi, functionParameters) {
```

Comment:
Utility function signature was simplified so that it does not need the hre (which makes tests simpler, and this function more efficient).